pkg/commands/internal/migrate/migrate_linters_settings.go

```
@@ -0,0 +1,1017 @@
+		Errorf:      old.Errorf,
+		ErrorfMulti: old.ErrorfMulti,
+		Asserts:     old.Asserts,
+		Comparison:  old.Comparison,
+	}
+
+	for _, allowedError := range old.AllowedErrors {
+		settings.AllowedErrors = append(settings.AllowedErrors, versiontwo.ErrorLintAllowPair{
+			Err: allowedError.Err,
+			Fun: allowedError.Fun,
+		})
+	}
+	for _, allowedError := range old.AllowedErrorsWildcard {
+		settings.AllowedErrorsWildcard = append(settings.AllowedErrorsWildcard, versiontwo.ErrorLintAllowPair{
+			Err: allowedError.Err,
+			Fun: allowedError.Fun,
+		})
+	}
+
+	return settings
+}
+
+func toExhaustiveSettings(old versionone.ExhaustiveSettings) versiontwo.ExhaustiveSettings {
+	return versiontwo.ExhaustiveSettings{
+		Check:                      old.Check,
+		DefaultSignifiesExhaustive: old.DefaultSignifiesExhaustive,
+		IgnoreEnumMembers:          old.IgnoreEnumMembers,
+		IgnoreEnumTypes:            old.IgnoreEnumTypes,
+		PackageScopeOnly:           old.PackageScopeOnly,
+		ExplicitExhaustiveMap:      old.ExplicitExhaustiveMap,
+		ExplicitExhaustiveSwitch:   old.ExplicitExhaustiveSwitch,
+		DefaultCaseRequired:        old.DefaultCaseRequired,
+	}
+}
+
+func toExhaustructSettings(old versionone.ExhaustructSettings) versiontwo.ExhaustructSettings {
+	return versiontwo.ExhaustructSettings{
+		Include: old.Include,
+		Exclude: old.Exclude,
+	}
+}
+
+func toFatcontextSettings(old versionone.FatcontextSettings) versiontwo.FatcontextSettings {
+	return versiontwo.FatcontextSettings{
+		CheckStructPointers: old.CheckStructPointers,
+	}
+}
+
+func toForbidigoSettings(old versionone.ForbidigoSettings) versiontwo.ForbidigoSettings {
+	settings := versiontwo.ForbidigoSettings{
+		ExcludeGodocExamples: old.ExcludeGodocExamples,
+		AnalyzeTypes:         old.AnalyzeTypes,
+	}
+
+	for _, pattern := range old.Forbid {
+		if pattern.Pattern == nil {
+			buffer, err := pattern.MarshalString()
+			if err != nil {
+				// impossible case
+				panic(err)
+			}
```

If this is impossible case, can we simply ignore an error?
```suggestion
			buffer, _ := pattern.MarshalString() // an error is impossible
```

---

because ignoring an error is not a good idea.